packages/components/src/Subscription/Component.purs

```diff
@@ -167,19 +167,30 @@ render self@{ props: props@{ subscription: sub@{ package } } } =
         where
           asyncWrapper = AsyncWrapper.asyncWrapper
             { wrapperState: self.state.wrapperProgress
-            , readyView: actionsContainer $ [ pauseIcon
-                                            , if maybe true Array.null self.state.pausedSubscriptions
-                                              then mempty
-                                              else removeSubscriptionPauses
-                                            , temporaryAddressChangeIcon
-                                            , deliveryReclamationIcon
-                                            ]
+            , readyView: actionsContainer $ defaultActions
             , editingView: identity
-            , successView: \msg -> successContainer [ DOM.div { className: "subscription--update-success check-icon" }, foldMap successMessage msg  ]
+            , successView: \msg -> actionsContainer $ append defaultActions $ [successWrapper msg]
             , errorView: \err -> errorContainer [ errorMessage err, tryAgain ]
```

Comment:
As far as I understood, we need to show the actions in case of error as well.

Reply:
Ah true, see latest commit ;)